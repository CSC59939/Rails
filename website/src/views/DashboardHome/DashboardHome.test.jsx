import React from 'react';
import renderer from 'react-test-renderer';
import {
  configure,
} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { DashboardHome } from '..';

configure({ adapter: new Adapter() });


describe('DashboardHome', () => {
  it('Render DashboardHome without any eror', () => { // eslint-disable-line no-undef
    expect(renderer.create(<DashboardHome />).toJSON()).toMatchSnapshot();
  });
});
