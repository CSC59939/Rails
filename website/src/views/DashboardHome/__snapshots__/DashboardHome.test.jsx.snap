// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashboardHome Render DashboardHome without any eror 1`] = `
<div
  className="DashboardHome"
>
  <div
    className="EventCalendarContainer"
  >
    <div
      className="rbc-calendar"
    >
      <div
        className="rbc-toolbar"
      >
        <span
          className="rbc-btn-group"
        >
          <button
            onClick={[Function]}
            type="button"
          >
            Today
          </button>
          <button
            onClick={[Function]}
            type="button"
          >
            Back
          </button>
          <button
            onClick={[Function]}
            type="button"
          >
            Next
          </button>
        </span>
        <span
          className="rbc-toolbar-label"
        >
          12/04/2018 â€” 01/03/2019
        </span>
        <span
          className="rbc-btn-group"
        >
          <button
            className=""
            onClick={[Function]}
            type="button"
          >
            Month
          </button>
          <button
            className="rbc-active"
            onClick={[Function]}
            type="button"
          >
            Agenda
          </button>
        </span>
      </div>
      <div
        className="rbc-agenda-view"
      >
        <span
          className="rbc-agenda-empty"
        >
          There are no events in this range.
        </span>
      </div>
    </div>
  </div>
</div>
`;
